docs/lib/graphqlapi/fragments/flutter/authz.md

```diff
@@ -0,0 +1,134 @@
+For client authorization AppSync supports API Keys, Amazon IAM credentials, Amazon Cognito User Pools, and 3rd party OIDC providers. This is inferred from the `amplifyconfiguration.json` file when you call `Amplify.configure()`.
```

Comment:
Shouldn't this say `amplifyconfiguration.dart` instead of `amplifyconfiguration.json`